elements/rh-card/docs/10-style.md

```diff
@@ -23,59 +23,63 @@ elements and styles can be placed inside.
 </uxdot-example>
 
 ## Theme
+All card variants are available in light and dark theme.
+
+### Light theme
 
 <uxdot-example width-adjustment="784px">
   <img src="{{ '../card-theme-light.svg' | url }}" alt="Card in light theme">
 </uxdot-example>
 
+### Dark theme
+
 <uxdot-example color-palette="darkest" width-adjustment="784px">
   <img src="{{ '../card-theme-dark.svg' | url }}" alt="Card in dark theme">
 </uxdot-example>
 
 ### Color
 
-Cards are secondary layouts that shouldn’t command too much attention and
-blend in with whatever background they’re placed on. The card container is the
-only required element and it consists of a background color and rounded
-corners. A thin border is required if the card background is the same color as
-the background it's placed on.
-
+Cards are secondary layouts that shouldn’t command too much attention and blend in with whatever background they’re placed on. The card container is the only required element and it consists of a background color, rounded corners, and a thin border.
 
 <div class="grid">
   <figure>
-    <uxdot-example width-adjustment="242px">
-      <img src="{{ '../card-color-light-white.svg' | url }}" alt="White card colors">
-    </uxdot-example>
+    <rh-card>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est egestas, a sollicitudin mauris tincidunt.</p>
+      <rh-cta href="#">Call to action</rh-cta>
+    </rh-card>
     <figcaption>
-      A white card with a light gray border is the most common use case in the
-      light theme
+      A light gray card without a border can also be used as an alternate option.
     </figcaption>
   </figure>
   <figure>
-    <uxdot-example width-adjustment="242px">
-      <img src="{{ '../card-color-light-gray.svg' | url }}" alt="Gray card colors">
-    </uxdot-example>
+    <rh-card color-palette="lighter">
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est egestas, a sollicitudin mauris tincidunt.</p>
+      <rh-cta href="#">Call to action</rh-cta>
+    </rh-card>
     <figcaption>
-      A light gray card without a border can also be used as an alternate option
+      A light gray card with a light gray border can also be used as an alternate option.
     </figcaption>
   </figure>
   <figure>
-    <uxdot-example color-palette="darkest" width-adjustment="242px">
-      <img src="{{ '../card-color-dark-black.svg' | url }}" alt="Black card colors">
-    </uxdot-example>
+    <rh-card color-palette="dark">
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est egestas, a sollicitudin mauris tincidunt.</p>
+      <rh-cta href="#">Call to action</rh-cta>
+    </rh-card>
     <figcaption>
-      A black card with a dark gray border is the most common use case in the
-      dark theme
+      A black card with a dark gray border is the most common use case in the dark theme.
     </figcaption>
   </figure>
```

Comment:
I wonder if for the dark cards here if we dont want to keep the `<uxdot-example color-palette="dark">` and place the card inside of it without setting its `color-palette`:

```Suggestion
<figure>
    <uxdot-example no-border color-palette="dark">
 <rh-card>
      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est egestas, a sollicitudin mauris tincidunt.</p>
      <rh-cta href="#">Call to action</rh-cta>
    </rh-card>
</uxdot-example>
    <figcaption>
      A black card with a dark gray border is the most common use case in the dark theme.
    </figcaption>
  </figure>
```

Screenshot of what this renders as:

![Screenshot 2024-06-27 at 3 00 09 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/13666/d3461573-c9c6-4bac-bb3c-c269acd8d7c2)

It's my thought that this more accurately represents the figcaption showing the grey border on a dark theme.

Reply:
Good suggestion